extensions/adobe/experience/analytics/experienceevent.schema.json

```diff
@@ -1640,405 +1639,405 @@
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop65":{
-           "title":"prop65",
-           "description":"Custom property variable65",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop66":{
-           "title":"prop66",
-           "description":"Custom property variable66",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop67":{
-           "title":"prop67",
-           "description":"Custom property variable67",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop68":{
-           "title":"prop68",
-           "description":"Custom property variable68",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop69":{
-           "title":"prop69",
-           "description":"Custom property variable69",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop70":{
-           "title":"prop70",
-           "description":"Custom property variable70",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop71":{
-           "title":"prop71",
-           "description":"Custom property variable71",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop72":{
-           "title":"prop72",
-           "description":"Custom property variable72",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop73":{
-           "title":"prop73",
-           "description":"Custom property variable73",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop74":{
-           "title":"prop74",
-           "description":"Custom property variable74",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
-        },
-        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop75":{
-           "title":"prop75",
-           "description":"Custom property variable75",
-           "$ref":"https://ns.adobe.com/experience/analytics/listdetails"
+        "https://ns.adobe.com/experience/analytics/custom-dimensions/list-props/prop1": {
```

Comment:
This whole custom property value thing bothers me!  It's not in line with the XDM model